keps/sig-cli/3104-introduce-kuberc/README.md

```diff
@@ -261,6 +262,12 @@ As a user I would like to be able to opt out of deprecation warnings.
 
 https://github.com/kubernetes/kubectl/issues/1317
 
+#### Story 5
```

Comment:
Do you think lock file is a prerequisite for beta promotion?

Reply:
https://github.com/kubernetes/kubernetes/pull/131619#pullrequestreview-2836473117

Reply:
This use case seems a little strange to me, are there other examples of behavior like this for customizations / aliases in other projects?

Reply:
I think, once we remove this story. This PR is ready for PRR.